pkgs/development/tools/build-managers/scons/4.5.2.nix

```diff
@@ -21,9 +25,13 @@ python3.pkgs.buildPythonApplication {
       --replace "build/doc/man/" ""
   '';
 
+  preConfigure = ''
+    python scripts/scons.py
+  '';
+
   postInstall = ''
-    mkdir -p "$out/share/man/man1"
-    mv "$out/"*.1 "$out/share/man/man1/"
+    mkdir -p "$man/share/man/man1"
+    mv "$out/"*.1 "$man/share/man/man1/"
```

Comment:
```suggestion
    installManPage $out/*.1
```

Reply:
I don't believe this is needed.

Reply:
Oh, yeah, that would only work if the man page wasn't already installed.